zhmcclient/_partition.py

```diff
@@ -773,3 +773,110 @@ def wait_for_status(self, status, status_timeout=None):
                     actual_status, statuses, status_timeout)
 
             time.sleep(1)  # Avoid hot spin loop
+
+    @logged_api_call
+    def increase_crypto_config(self, crypto_adapters,
+                               crypto_domain_configurations):
+        """
+        Adds the specified adapters and/or domain configurations
+        to the crypto configuration of the corresponding partition.
+
```

Comment:
Several issues with this paragraph:

1. "and/or" is not correct. It adds the adapters *and* the domain configurations.  In fact, it is even more complex because the previously assigned adapters also count.
2. "of the corresponding partition." is a bit strange because this is a method on a partition object, and we use "this partition" in the description of other methods.

To address all of that, I suggest replacing this paragraph with:
```
        Adds the specified adapters and domain configurations to the crypto
        configuration of this partition.

        The resulting crypto configuration of the partition always has all specified
        domains allocated on all specified crypto adapters, including the
        domains and adapters that were part of the previous crypto configuration
        of the partition.```
plus to add an example:
```
        Example:
            Specifying domain configurations with domain indexes 2, 3, and 4
            on crypto adapters A and B will result in a crypto configuration